### PR TITLE
Fix name to Carina

### PR DESCRIPTION
**Pull request by jnoller**
Fixing citations of Rackspace w/ Carina

### CODE REVIEW

File: _tutorials/002-docker-101.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Docker 101
 slug: docker-101-introduction-docker
-description: Introduction to Docker, why Docker is used with Rackspace Container Services, and instructions on how to install and use Docker.
+description: Introduction to Docker, why Docker is used with Carinas, and instructions on how to install and use Docker.
```
Comment (rgbkrk) [inline _tutorials/002-docker-101.md]: s/Carinas/Carina

File: _tutorials/002-docker-101.md [hunk 2]
```diff
@@ -51,9 +51,9 @@ Docker containers contain an image, a set of standard operations, and an executi
 
 <!--diagram explaining Docker containers here-->
 
-### Docker and Rackspace Container Service
+### Docker and Carina
 
-Because Docker containers are interchangeable, when an application is deployable with containers, a host is needed in order to take your build from local to scalable. Rackspace Container Service(RCS) takes your containers and deploys them using the Rackspace Managed Cloud. Both of these services combined allow you to deploy even though most complex architectures with ease.
+Because Docker containers are interchangeable, when an application is deployable with containers, a host is needed in order to take your build from local to scalable. Carina takes your containers and deploys them using the Carina. Both of these services combined allow you to deploy even though most complex architectures with ease.
```
Comment (rgbkrk) [inline _tutorials/002-docker-101.md]: "deploys them using the Carina". Wow this search/replace had me in tears. File this under "paragraph needs changing in general".

**Comment by rgbkrk** [comment]
Some of these had URL changes that will require changes in filenames too.

**Comment by jnoller** [comment]
@rgbkrk I looked in _tutorials but missed that.

File: _tutorials/getting-started-on-carina.md [hunk 3]
```diff
@@ -23,8 +23,8 @@ A cluster is a pool of compute, storage, and networking resources that serves as
 
 To create your cluster, perform the following steps:
 
-1. Sign in to [http://mycluster.rackspacecloud.com](http://mycluster.rackspacecloud.com).
-1. Enter a cluster name in the **Create New** field. For example: `mycluster`
+1. Sign in to [http://app.getcarina.com.rackspacecloud.com](http://app.getcarina.com.rackspacecloud.com).
```
Comment (rgbkrk) [inline _tutorials/getting-started-on-carina.md]: This... might be wrong.

**Pull request closed without merging**